pandas/tests/tslibs/test_parsing.py

```diff
@@ -24,7 +25,8 @@ def test_parse_time_string():
 
 def test_parse_time_string_invalid_type():
     # Raise on invalid input, don't just return it
-    with pytest.raises(TypeError):
+    msg = re.escape("Argument 'arg' has incorrect type (expected str, got tuple)")
+    with pytest.raises(TypeError, match=msg):
```

Comment:
for me, because the match parameter expects a regex and the msg is the message string. I would tend to prefer the msg not escaped and then match=re.escape(msg)

Reply:
Thanks for the feedback... I have sorted it out.